### ISSUE TITLE
Compilation error

### ISSUE THREAD

**Issue by sikmir** [gopasspw/gopass#2280]
### Summary
Compilation failed with go 1.18:
```
...
gopass> github.com/gopasspw/gopass/internal/backend
gopass> # github.com/gopasspw/gopass/internal/backend
gopass> internal/backend/registry.go:85:45: internal compiler error: assertion failed
...
```

### Environment
- OS: NixOS
- gopass Version: 1.14.3

**Comment by dominikschulz** [comment]
That's a bug in Go 1.18.4.

Fixed at head.

**Comment by sikmir** [comment]
Could you please advise, is there any simple workaround I can apply as local patch to get it compiled with go 1.18.4:
https://github.com/gopasspw/gopass/blob/832d2cde4d5a93721c291644107b3a89acb7e741/internal/backend/registry.go#L85
Since it will be fixed only in go 1.18.5.
